docs/tutorials/tabular_prediction/tabular-faq.md

```diff
@@ -0,0 +1,22 @@
+# FAQ
+:label:`sec_faq`
+
+How can I perform inference on a file that won't fit in memory?
+
+The Tabular Dataset API works with pandas Dataframes, which supports chunking data in to sizes that fit in memory.
+Here's an example of one such chunk-based inference:
+
+```{.python .input}
+train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+predictor = task.fit(train_data=train_data)
+
+r = requests.get('https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv', allow_redirects=True)
+open('test.csv', 'wb').write(r.content)
+reader = pd.read_csv('test.csv', chunksize=32)
+for chunk in reader:
+    test_data = task.Dataset(df=chunk)
+    predictor.evaluate(test_data)
```

Comment:
It might be better for it to show preds as a list of Series instead, which are finally concatenated to give the same result as if you did predictor.predict(full_test_data).

```
reader = pd.read_csv(path_test, chunksize=1000)
y_pred = []
y_true = []
for df_chunk in reader:
    y_pred.append(predictor.predict(df_chunk, as_pandas=True))
    y_true.append(df_chunk['class'])
y_pred = pd.concat(y_pred, axis=0, ignore_index=True)
y_true = pd.concat(y_true, axis=0, ignore_index=True)
predictor.evaluate_predictions(y_true=y_true, y_pred=y_pred)
```